lib/Dockerfile

```diff
@@ -14,7 +14,7 @@
 
 # Dockerfile for Ignite
 
-FROM apacheignite/ignite:2.8.1
+FROM apacheignite/ignite:2.10.0
 
 MAINTAINER Liguang Xie <lxie@futurewei.com>
 
```

Comment:
Jenkins uses lib/ignite.Docker file.  It needs to be changed.

Reply:
> apache ignite 2.10.0 is not available as zip file. It is available as .rpm file. Perhaps moving to 2.11.0, which is available as zip file and compatible with 2.9 is better option.

2.10.0 zip is here: https://ignite.apache.org/download.cgi
2.11.0 is just released two weeks ago. Let us wait some time to let it cool down.